src/functions/src/main/java/org/apache/jmeter/functions/DigestEncodeFunction.java

```diff
@@ -74,19 +74,16 @@
     public String execute(SampleResult previousResult, Sampler currentSampler) throws InvalidVariableException {
         String digestAlgorithm = values[0].execute();
         String stringToEncode = values[1].execute();
-        String salt = null;
-        if (values.length > 2) {
-            salt = values[2].execute();
-        }
+        String salt = values.length > 2 ? values[2].execute() : null;
         String encodedString = null;
         try {
             MessageDigest md = MessageDigest.getInstance(digestAlgorithm);
-            md.update(stringToEncode.getBytes(UTF_8));
+            md.update(stringToEncode.getBytes(StandardCharsets.UTF_8.name()));
```

Comment:
Can `stringToEncode.getBytes(StandardCharsets.UTF_8)` be used here?
What is the point of `.name()`?